src/scss/variables/_colors.scss

```diff
@@ -1,3 +1,13 @@
 
 $wolox-green: #8dc63f;
 $wolox-blue: rgba(85, 164, 218, .7);
+$background: #FDFDF9;
```

Comment:
the variable names should be related to the color and not to the place they are used in, this is because you might want to use the same color in different places in the future and it would be awkward to have a background-color: $text-color